### PR TITLE
Don't dump exception to console when repo URI isn't in format expected

### PR DESCRIPTION
**Pull request by JohnTortugo**
- Check that the repo exist before checking that the branch exist
- Patch some methods to false if repo/branch doesn't exist AND prompt == false.

### CODE REVIEW

File: src/Microsoft.DotNet.Darc/src/Darc/Helpers/UxHelpers.cs [hunk 1]
```diff
@@ -275,6 +280,7 @@ public static async Task<bool> VerifyAndConfirmBranchExistsAsync(IRemote remote,
                 {
                     return PromptForYesNo("Continue?");
                 }
+                return false;
```
Comment (JohnTortugo) [inline src/Microsoft.DotNet.Darc/src/Darc/Helpers/UxHelpers.cs]: This wasn't part of the issue but I think we should return false here if an exception was thrown and prompt == false. Right?

Comment (riarenas) [reply src/Microsoft.DotNet.Darc/src/Darc/Helpers/UxHelpers.cs]: I don't think so, we default to creating what you asked to be created if you pass --quiet, even if it's something that doesn't exist right now.

(I personally like that behavior, sometimes I create a subscription or create a default channel and afterwards push the branch)

Comment (JohnTortugo) [reply src/Microsoft.DotNet.Darc/src/Darc/Helpers/UxHelpers.cs]: Ok. I'm going to revert that.

**Comment by JohnTortugo** [comment]
Sample outputs below.

```
PS C:\tmp\arcade-services> darc set-repository-policies --repo dotnet/arcade --standard-automerge --branch release/0.10.0-preview777  --quiet
Unknown repository 'dotnet/arcade', repo type set to 'None'. Please inform the full URL of the repository including the http[s] prefix.
The target repository 'dotnet/arcade' doesn't have a Maestro installation. Aborting merge policy creation.

PS C:\tmp\arcade-services> darc set-repository-policies --repo http://github.com/xyz/dotnet/arcade --standard-automerge --branch release/0.10.0-preview777  --quiet
The target repository 'http://github.com/xyz/dotnet/arcade' doesn't have a Maestro installation. Aborting merge policy creation.

PS C:\tmp\arcade-services> darc set-repository-policies --repo https://github.com/dotnet/arcade --standard-automerge --branch release/0.10.0-preview777  --quiet
Warning: Could not find an eng/Version.Details.xml at 'https://github.com/dotnet/arcade@release/0.10.0-preview777'. Dependency updates may not happen as expected.
Successfully updated merge policies for https://github.com/dotnet/arcade@release/0.10.0-preview777.

PS C:\tmp\arcade-services> darc set-repository-policies --repo https://github.com/dotnet/arcade --standard-automerge --branch release/0.10.0-preview777
Warning: Could not find an eng/Version.Details.xml at 'https://github.com/dotnet/arcade@release/0.10.0-preview777'. Dependency updates may not happen as expected.
Continue? (y/n) y
Successfully updated merge policies for https://github.com/dotnet/arcade@release/0.10.0-preview777.

PS C:\tmp\arcade-services> darc set-repository-policies --repo dotnet/arcade --standard-automerge --branch release/0.10.0-preview777
Unknown repository 'dotnet/arcade', repo type set to 'None'. Please inform the full URL of the repository including the http[s] prefix.
The target repository 'dotnet/arcade' doesn't have a Maestro installation. Aborting merge policy creation.
```

**Comment by JohnTortugo** [comment]
@riarenas - pinging in case you want to take a look again. BTW, I know you're oof/wfh, no hurry here.

File: src/Microsoft.DotNet.Darc/src/Darc/Helpers/LocalSettings.cs [hunk 2]
```diff
@@ -116,10 +116,13 @@ public static DarcSettings GetDarcSettings(CommandLineOptions options, ILogger l
                                 options.AzureDevOpsPat;
                         }
                     }
-
+
```
Comment (riarenas) [inline src/Microsoft.DotNet.Darc/src/Darc/Helpers/LocalSettings.cs]: [nit] added trailing spaces

**Pull request merged**